api/src/dtos/listings/listing-update.dto.ts

```diff
@@ -157,4 +162,9 @@ export class ListingUpdate extends OmitType(Listing, [
   @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
   @Type(() => IdDTO)
   requestedChangesUser?: IdDTO;
+
+  @Expose()
+  @ApiPropertyOptional()
+  @IsBoolean({ groups: [ValidationsGroupsEnum.default] })
+  isVerified?: boolean;
```

Comment:
chore: This change is not necessary, values to `listing-update.dto` and `listing-create.dto` only need to be added if the value is different than `listing.dto.ts` or it doesn't exist there. So you can delete it from this file.